lms/tasks/email_digests.py

```diff
@@ -56,7 +57,7 @@ def send_instructor_email_digest_tasks(*, batch_size):  # pylint:disable=unused-
                     # Note here that we are considering earlier launches
                     # We rather take a few more courses that miss some cases
                     # where the launch that originated the annotations was made around the cutoff time.
-                    Event.timestamp >= created_after - timedelta(days=7),
+                    Event.timestamp >= created_before - timedelta(days=14),
```

Comment:
This is an optimisation where we only consider instructors whose courses have had events (such as assignment launches) in the last seven days. Previously all emails covered only a 24hr period but now a single email can count annotations over up to a week-long period, so increasing the seven days here to 14 to make sure we don't miss any instructors.

Also basing it off `created_before` since there's no longer a `created_after` in this method.

Reply:
We'll have to keep an eye on how this performs with the bigger time window.